operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-statefulset.yaml

```diff
@@ -166,6 +166,13 @@ spec:
             {{- with .Values.store_gateway.env }}
               {{- toYaml . | nindent 12 }}
             {{- end }}
+            {{- $cpu_request := dig "requests" "cpu" nil .Values.store_gateway.resources }}
+            {{- if $cpu_request }}
+              {{- $cpu_request_doubled := include "mimir.parseCPU" (dict "value" $cpu_request) | float64 | mulf 2 | ceil }}
+              {{- $cpu_request_plus_four := include "mimir.parseCPU" (dict "value" $cpu_request) | float64 | addf 4 | ceil }}
+            - name: "GOMAXPROCS"
+              value: {{ max $cpu_request_doubled $cpu_request_plus_four | toString | toYaml }}
```

Comment:
Why is `toYaml` needed?

Reply:
It shouldn't be required because the expected output value is always an integer number, but it's just a safe protection to ensure the string is correctly escaped. We do the same for GOMEMLIMIT.